Base/usr/share/man/man1/su.md

```diff
@@ -1,41 +1,42 @@
 ## Name
 
-su - run a command with substitute user
+su - switch to another user
 
 ## Synopsis
 
-```
-$ su [user]
+```**sh
+$ su [--help] [--version] <user>
 ```
 
 ## Description
 
-**su** allows commands to be run with a substitute user.
-
-When called with no user-specified, **su** defaults to running an interactive shell as *root*. 
+`su` switch to another user.
```

Comment:
This could be worded better.

Reply:
How about `su` switch to another specified user?

Reply:
Both are not good English sentences.

> `su`: Switch to another user.

Reply:
How about `su` switch to a particular user?

Reply:
The point is that for proper English, we need a colon here.

Reply:
So, it should be: `su` Switch to another user?

Reply:
Do you not understand what a colon `:` is?